File: app/src/main/java/quangnam/com/sample/base/rxjava/BaseSuccessAction.java
package quangnam.com.sample.base.rxjava;

import io.reactivex.functions.Consumer;

/**
 * Created by quangnam on 2/19/18.
 * Project Sample
 */

public class BaseSuccessAction<T> implements Consumer<T> {
    @Override
    public void accept(T t) throws Exception {
        return;
    }
}

File: app/src/main/java/quangnam/com/sample/base/rxjava/BaseSuccessAction.kt
package quangnam.com.sample.base.rxjava

import io.reactivex.functions.Consumer

/**
 * Created by quangnam on 2/19/18.
 * Project Sample
 */

open class BaseSuccessAction<T> : Consumer<T> {
    @Throws(Exception::class)
    override fun accept(t: T) {
        return
    }
}
